packages/patternfly-4/react-core/src/components/AboutModal/AboutModalBoxHero.js

```diff
@@ -14,9 +16,14 @@ const defaultProps = {
   className: ''
 };
 
-const AboutModalBoxHero = ({ className, ...props }) => (
-  <div {...props} className={css(styles.aboutModalBoxHero, className)} />
-);
+const AboutModalBoxHero = ({ className, bgImgSrc, ...props }) => {
+  const bgStyle = (bgImgSrc !== '') ? {
+    "--pf-c-about-modal-box__hero--sm--BackgroundImage": `url(${bgImgSrc})`
```

Comment:
Thinking we should make "--pf-c-about-modal-box__hero--sm--BackgroundImage" a constant stored in utils constant.   Maybe make a CSS section where we can add one offs like this that could be affected by a change in core down the road.  This way if we have to change this name with a core update we can find it easier.

Reply:
I believe you can
`import { c_about_modal_box__hero_sm_BackgroundImage } from '@patternfly/react-tokens'`
and `c_about_modal_box__hero_sm_BackgroundImage.name` gives you the variable name

Reply:
If that works then let's go that route.